f600233d1b154d507b782f84604bcb12/update/sys_dictionary_x_g_dis_atat_contacts_formal_name.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update>
+    <sys_dictionary action="INSERT_OR_UPDATE" element="formal_name" table="x_g_dis_atat_contacts">
+        <active>true</active>
+        <array>false</array>
+        <array_denormalized>false</array_denormalized>
+        <attributes/>
+        <audit>false</audit>
+        <calculation><![CDATA[// build rank or salutation (role dependent)
+function buildRankOrSalutation(current) {
+    var output = '';
+    //include rank for MIL
+    if (current.role == 'MILITARY') {
```

Comment:
Should we be using the strict `===` and `!==`?

Reply:
Good question.  I'll try it.